apps/indexer/lib/indexer/fetcher/token_instance/retry.ex

```diff
@@ -32,31 +34,35 @@ defmodule Indexer.Fetcher.TokenInstance.Retry do
       Chain.stream_token_instances_with_error(
         initial_acc,
         fn data, acc ->
-          reducer.(data, acc)
+          reduce_if_que_is_not_full(data, acc, reducer)
         end
       )
 
     acc
   end
 
+  defp reduce_if_que_is_not_full(data, acc, reducer) do
```

Comment:
shouldn't it be named like that?
```suggestion
  defp reduce_if_queue_is_not_full(data, acc, reducer) do
```

Reply:
Nice catch, fixed in ce1de0aa5424a0a32eee08c7fe68d271a8643da3